tests/test_requests.py

```diff
@@ -910,7 +927,7 @@ def test_certificate_failure(self, httpbin_secure):
         """
         When underlying SSL problems occur, an SSLError is raised.
         """
-        with pytest.raises(SSLError):
+        with pytest.raises(RequestsSSLError):
```

Comment:
Does this mean that previously Requests wasn't wrapping SSLErrors? If this one is being raised on connect instead of read I wonder if this would be a big break for users?